### ISSUE TITLE
[Feature Request] "Swap screen" should also change "Top only" screen view to "Bottom only" (and vice-versa)

### ISSUE THREAD

**Issue by mark21gt** [Arisotura/melonDS#1047]
When using the "Even", "Emphasize top" or "Emphasize bottom" display modes, the "Swap screen" option does its job by swapping the two screens that are being currently shown by the emulator.

It would be really nice if this option could also swap between the "Top only" and "Bottom only" display modes (or maybe a new mode called "Single screen" that would allow the screen swap could be added). 

By doing this, the emulator could display only one screen at a time, while still allowing it to be changed by using the "Swap screen" shortcut.

Using the "Swap screen" function when using these two modes ("Top only" and "Bottom only") currently has no effect.

Adding this option would give a great improvement in presentation for games which don't require both screens at the same time and the second screen can get distracting or awkward when being shown side-by-side while not being used (especially in RPGs).

**Comment by Slider-Whistle** [comment]
Agreed. The "Swap Key" keybinding should work while in single-screen (top/bottom screen-only) mode.

**Comment by RSDuck** [comment]
fixed with #1283
